src/stateless/slider.vue

```diff
@@ -0,0 +1,597 @@
+<template>
+    <div class="slider-container" :class="Object.assign(sizeClass(), themeClass())">
+
+        <div class="slider-header">
```

Comment:
Use BEM, block name should probably be `slider`, so this should be `slider__header`